implementation.md

```diff
@@ -0,0 +1,22 @@
+# Implementation Recommendations
+
+This document describes implementation recommendations for a STAC API.
+
+## CORS
+
+It is recommended that public APIs advertise a permissive CORS configuration so UIs running on a different domain
```

Comment:
Maybe add a short sentence about what CORS is useful for (JS in Browser)? UIs in general is a bit misleading, i.e. it would not apply to a Java Desktop UI. Or link to a CORS article (e.g. wikipedia).

Reply:
I made this more clear.